Source/Scene/ModelExperimental/ModelExperimentalSceneGraph.js

```diff
@@ -450,6 +494,43 @@ ModelExperimentalSceneGraph.prototype.updateModelMatrix = function () {
   }
 };
 
+const scratchObjectSpace = new Matrix4();
+
+ModelExperimentalSceneGraph.prototype.updateJointMatrices = function () {
+  const model = this._model;
+  const skinnedNodes = model._skinnedNodes;
+
+  for (let i = 0; i < skinnedNodes.length; i++) {
+    const node = skinnedNodes[i];
+
+    const objectSpace = Matrix4.inverseTransformation(
```

Comment:
If I'm reading this correctly, `objectSpace` corresponds to `nodeToRoot^(-1)` in the matrix equation below? we should rename one of them so they match. 

... or maybe rename this matrix `rootToNode`? simpler than `inverseNodeToRoot`

Reply:
whoops, that was an artifact of copying `Model.js` code. thanks for the catch